### PR TITLE
JSON serialization fixes + cleanup

### PR DESCRIPTION
**Pull request by BalassaMarton**

### CODE REVIEW

File: netcore-webview-test/MainWindow.xaml.cs [hunk 1]
```diff
@@ -44,21 +44,23 @@ async void MessageReceived(object sender, CoreWebView2WebMessageReceivedEventArg
 
         private class PsMessage
         {
-            public int msgId { get; set; }
-            public string action { get; set; }
-            public string data { get; set; }
+            public int msgId;
```
Comment (BalassaMarton) [inline netcore-webview-test/MainWindow.xaml.cs:47]: I wasn't aware that field support was already added :)

File: ws-test/Startup.cs [hunk 2]
```diff
@@ -113,29 +107,36 @@ ValueTask SendMessage(ReadOnlyMemory<byte> message)
                 }
             }
 
-            await webSocket.CloseAsync(webSocket.CloseStatus.Value, webSocket.CloseStatusDescription, CancellationToken.None);
```
Comment (BalassaMarton) [inline ws-test/Startup.cs:116]: Not sure about this one (it's working but the doc says we have to check for idle clients and close the socket manually)

File: netcore-webview-test/MainWindow.xaml.cs (hunk 1 above)
Comment (Sly1024) [reply netcore-webview-test/MainWindow.xaml.cs:47]: yeah, me neither. I did the same thing before your PR, just went with the properties. Good to know though.
This is just a PoC, doesn't have to be prod quality, and we'll probably let SignalR do the low level work. But it's good to see the performance we can get with raw websocket.

Comment (BalassaMarton) [reply netcore-webview-test/MainWindow.xaml.cs:47]: We should still keep ws on the table. It's more accessible to app developers than signalr.

**Pull request merged**